### DISCUSSION TITLE
directly selecting the content of comments

### DISCUSSION THREAD

**Discussion by aecra** [Docile-Alligator/Infinity-For-Reddit#854] [Ideas]
When viewing comments in an unskilled language, many clicks are required to copy and then translate the comment.  And if you can provide the function of direct selection, I can use the Google Translate app to open the translation floating window directly on the current page.

**Comment by Wladefant** [comment]
I would not justify this request with the translate reason, but rather with the reason that you normally do not need the whole message for copying.

**Reply by KaKi87** [reply]
Agreed.

Any news on this ?

Thanks

**Comment by vhscom** [comment]
Telegram, for example, allows text selection after by a user long-presses a card. For the translation use case I would prefer something integrated, such as a setting to input an API key to enable a translator using a web service. The native Reddit client has something similar to this. The easiest mitigation right now may be to reduce the number of interactions required to copy a post to the clipboard.
